aspnetcore/release-notes/aspnetcore-1.1.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: What's new in ASP.NET Core 1.1
 author: rick-anderson
+ms.author: riande
```

Comment:
Nit: Move `ms.author` below `monikerRange` in all 3 of these docs. @guardrex sorted all of these alphabetically a while back.